src/components/application_manager/src/commands/mobile/set_global_properties_request.cc

```diff
@@ -393,7 +394,19 @@ void SetGlobalPropertiesRequest::PrepareUIRequestMenuAndKeyboardData(
   if (is_keyboard_props_present) {
     out_params[hmi_request::keyboard_properties] =
         msg_params[hmi_request::keyboard_properties];
-    app->set_keyboard_props(msg_params[hmi_request::keyboard_properties]);
+
+    const bool is_auto_complete_text_present =
+        out_params[hmi_request::keyboard_properties].keyExists(
+            strings::auto_complete_text);
+    const bool is_auto_complete_list_present =
+        out_params[hmi_request::keyboard_properties].keyExists(
+            strings::auto_complete_list);
+    if (is_auto_complete_list_present && is_auto_complete_text_present) {
+      out_params[hmi_request::keyboard_properties].erase(
+          strings::auto_complete_text);
+    }
```

Comment:
@LuxoftAKutsan, @dtrunov Ok I would prefer to remove my new field from "duplicate names from hmi_request" however I think that this group of duplicate fields is a bit confusing, maybe I should delete the rest two fields in it too? (I can make it as a separate commit)